client/landing/stepper/declarative-flow/internals/steps-repository/import/helper.ts

```diff
@@ -45,6 +46,12 @@ export function getFinalImporterUrl(
 		} );
 	}
 
+	if ( migrateEntireSiteFlow ) {
```

Comment:
I'm not very happy with this new argument here. It seems it shouldn't be part of the import, but I couldn't think any other approach for this.